src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -147,6 +149,36 @@ Scalar<DataVector> ricci_scalar(
 
   return ricci_scalar;
 }
+
+template <typename Frame>
+Scalar<DataVector> area_element(
+    const tnsr::ii<DataVector, 3, Frame>& lower_spatial_metric,
+    const StrahlkorperTags::StrahlkorperTags_detail::Jacobian<Frame>& jacobian,
+    const tnsr::i<DataVector, 3, Frame>& normal_one_form,
+    const DataVector& radius,
+    const tnsr::i<DataVector, 3, Frame>& r_hat) noexcept {
+  auto Theta = make_with_value<tnsr::I<DataVector, 3, Frame>>(r_hat, 0.0);
+  auto Phi = make_with_value<tnsr::I<DataVector, 3, Frame>>(r_hat, 0.0);
+
+  for (size_t i = 0; i < 3; ++i) {
+    Theta.get(i) = radius * jacobian.get(i, 0);
+    Phi.get(i) = radius * jacobian.get(i, 1);
+    for (size_t j = 0; j < 3; ++j) {
+      Theta.get(i) += r_hat.get(i) * radius *
+                      (r_hat.get(j) - normal_one_form.get(j)) *
+                      jacobian.get(j, 0);
+      Phi.get(i) += r_hat.get(i) * radius *
+                    (r_hat.get(j) - normal_one_form.get(j)) *
+                    jacobian.get(j, 1);
+    }
+  }
+  auto area_element = make_with_value<Scalar<DataVector>>(r_hat, 0.0);
```

Comment:
initialize `area_element` to `square(radius)` , remove `radius` from the loops above, and then in the next statement do `*=` instead of `=`